PaddleRec/ctr/deepfm/README.md

```diff
@@ -15,7 +15,7 @@ This model implementation reproduces the result of the paper "DeepFM: A Factoriz
 ```
 
 ## Environment
-- PaddlePaddle 1.6
+- **PaddlePaddle version >= 1.6, or develop version**
 
```

Comment:
**Now all models in PaddleRec require PaddlePaddle version 1.6 or higher, or suitable develop version.**

Reply:
solved